draft-ietf-rtcweb-jsep.xml

```diff
@@ -1161,7 +1161,18 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
                 </section>
 
                 <section title="addTransceiver" anchor="sec.addTransceiver">
-                  <t>[TODO]</t>
+                  <t>
+                    The addTransceiver method adds a transceiver to the PeerConnection.
+                    If a MediaStreamTrack argument is provided, then the transceiver will
+                    be configured with that media type and attached to that track. Otherwise,
+                    the application MUST explicitly specify the type; this mode is useful
+                    for creating recvonly transceivers as well as for creating transceivers
+                    which will be attached to a track at some later point.
+                    At the time of creation, the application can also specify a transceiver
+                    direction attribute, a set of MediaStreams which the transceiver is
+                    associated with (allowing LS group assignments),
+                    and a set of encodings for the media (used for simulcast [TODO section]).
```

Comment:
Do you want to expand on how the direction and MediaStreams are used? I think spelling this out would be useful.